README.md

```diff
@@ -110,7 +110,7 @@ If you are interested in fixing issues and contributing directly to the code bas
 
 ### 1. Prerequisites
 
-Ensure you have following components:
+Ensure you have installed following tools:
```

Comment:
```diff
- Ensure you have following components:
+ Ensure you have installed following tools:
```
should have a 'the' inbetween the 'following tools' and 'installed' due to the fact that 'tool' is a noun.